apps/site/test/site/lib/schedule_note_test.exs

```diff
@@ -36,6 +36,16 @@ defmodule Site.ScheduleNoteTest do
       )
     end
 
+    test "returns a schedule note with hours of operation for all lines" do
+      @all_lines
+      |> Enum.map(&ScheduleNote.new/1)
+      |> Enum.map(
+        assert(fn note ->
+          note.hours_of_operation
```

Comment:
Is there a more detailed assertion we can make here? (that would nevertheless be resistant to changes in live data...)